packages/cli/src/commands/join.ts

```diff
@@ -418,14 +419,22 @@ export async function handleJoin(argv: JoinArgv, packageVersion: string) {
         joinedDef.paths[path].parameters = [];
       }
 
-      for (const parameter of pathItem.parameters as Oas3Parameter[]) {
+      for (const parameter of pathItem.parameters as (Oas3Parameter & { $ref: string })[]) {
```

Comment:
Shouldn't it be `Oas3Parameter | { $ref: string }` instead of the `&`?
Can you make it more general? I mean, the same logic could be applied to other parts of a definition, not only to path parameters.

Reply:
It looks that we don't need this logic anywhere else